### PR TITLE
Multilanguage Spanish: Reviewed all text strings

### PR DESCRIPTION
**Pull request by enteropositivo**
## Description

I've reviewed all the text strings in Spanish and rewritten some of them to fit the width of the screen and make them properly understood.    Two python files have also been modified to add strings not contemplated by Babel

### CODE REVIEW

File: src/seedsigner/views/tools_views.py [hunk 1]
```diff
@@ -458,10 +459,10 @@ class ToolsAddressExplorerSelectSourceView(View):
 
     def run(self):
         seeds = self.controller.storage.seeds
-        self.SCAN_SEED = ("Scan a seed", SeedSignerIconConstants.QRCODE)
-        self.SCAN_DESCRIPTOR = ("Scan wallet descriptor", SeedSignerIconConstants.QRCODE)
-        self.TYPE_12WORD = ("Enter 12-word seed", FontAwesomeIconConstants.KEYBOARD)
-        self.TYPE_24WORD = ("Enter 24-word seed", FontAwesomeIconConstants.KEYBOARD)
+        self.SCAN_SEED = _("Scan a seed", SeedSignerIconConstants.QRCODE)
```
Comment (jdlcdl) [inline src/seedsigner/views/tools_views.py:462]: These were a tuple previously (having a string and constant), and now only the string needs to be replaced with the gettext call.  ie:
`self.SCAN_SEED = (_("Scan a seed"), SeedSignerIconConstants.QRCODE)`

**Comment by jdlcdl** [comment]
I've temporarily altered a total of 4 lines very similar to the code comment above.

As well, if you are making i18n changes to code, you can also submit the master ./babel/messages.pot if you'd like... since it has likely been done at your end during development, that way I won't mess with comments and can update all langs once merged.

I'm still writing up what I have to do as "iteration" work whenever i18n/l10n additions are made, so I'll have a better idea and be sure myself once I've been thru this process once.

If you like, feel free to add more commits, else I'll comment below here as I figure this out and make sure all is working... before merging.

Thank you enteropositivo!

**Comment by jdlcdl** [comment]
I've made a commit to my clone of seedsigner-screenshots [here](https://github.com/jdlcdl/seedsigner-screenshots/blob/v0.7.0_MultiLanguage/es/README.md).  I believe it to be representative of your changes, but please let me know.

**Comment by enteropositivo** [comment]
I have to check some screens with the emulator, because I am seeing in your screenshots that although the translations are in the .po file, they are not being reflected on the screens

**Comment by jdlcdl** [comment]
and I'm still organizing at my end.  I have not run this through my development seedsigner... because it's still "building wheel for numpy" which takes ages.  I generated the screenshots "blind", so I really have no clue what I may have broken or forgotten to do.

**Comment by jdlcdl** [comment]
This seems to be mostly working for me.  Exceptions are:
* green toast for "sdcard" messages are not being localized because they're in a thread that isn't locale aware... needs a fix.
* I believe that "Misc Error Views" would be localized too, except that when the settings screenshots run, the locale got changed... needs a fix.
Update: I'll change tests/screenshot_generator/generator.py#127 to an f-string so it can pickup "locale={target_locale}" to fix this.

Otherwise, random translation changes that I check on my pi0 dev system are showing translated... while some phrases still over-run screen size (which is a known issue since Keith's early work -- "needs scrolling!").

**Pull request merged**